main.go

```diff
@@ -136,6 +140,12 @@ func cmd() *cobra.Command {
 		},
 		Run: func(_ *cobra.Command, args []string) {
 			err := upgrade.UpgradeProvider(context, repoOrg, repoName)
+			if err != nil {
+				msg, err := createFailureIssue(context, repoOrg, repoName, err.Error())
```

Comment:
Probably want to format this error message to include a link to the workflow that failed.

I think we can access that via a GitHub env var.